solutions/agents/variables.tf

```
@@ -174,42 +174,42 @@ variable "cloud_monitoring_metrics_filter" {
-    error_message = "Invalid input for `cloud_monitoring_metrics_filter`. Valid options for 'type' are: `include` and `exclude`. If empty, no metrics are included or excluded."
+    error_message = "The specified `type` for the `cloud_monitoring_metrics_filter` is not valid. Specify either `include` or `exclude`. If the value for `type` is not specified, no metrics are included or excluded."
   }
 }
 
 variable "cloud_monitoring_agent_tags" {
   type        = list(string)
-  description = "List of tags to associate to all matrics that the agent collects. NOTE: Use the 'cloud_monitoring_add_cluster_name' variable to add the cluster name as a tag."
+  description = "A list of the tags to associate with the metrics that the IBM Cloud Monitoring agent collects. To add the cluster name as a tag, use the `cloud_monitoring_add_cluster_name` variable."
   default     = []
   nullable    = false
 }
 
 variable "cloud_monitoring_add_cluster_name" {
   type        = bool
-  description = "If true, configure the cloud monitoring agent to attach a tag containing the cluster name to all metric data."
+  description = "When set to `true`, this variable configures the IBM Cloud Monitoring agent to attach a tag that contains the cluster name to all log messages."
   default     = true
 }
 
 variable "cloud_monitoring_agent_name" {
-  description = "Cloud Monitoring agent name. Used for naming all kubernetes and helm resources on the cluster. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'"
+  description = "The name of the IBM Cloud Monitoring agent that is used to name the Kubernetes and Helm resources on the cluster. If a prefix input variable is passed, the name of the IBM Cloud Monitoring agent is prefixed to the value in the `<prefix>-value` format."
   type        = string
   default     = "sysdig-agent"
 }
 
 variable "cloud_monitoring_agent_namespace" {
   type        = string
-  description = "Namespace where to deploy the Cloud Monitoring agent. Default value is 'ibm-observe'."
+  description = "The namespace to deploy the IBM Cloud Monitoring agent in. The default value of the namespace is `ibm-observe`."
   default     = "ibm-observe"
   nullable    = false
 }
 
 variable "cloud_monitoring_agent_tolerations" {
-  description = "List of tolerations to apply to Cloud Monitoring agent. Default value means that it will match any taint on any node except the master node. See https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/ for more info."
+  description = " The list of tolerations to apply to the IBM Cloud Monitoring agent. The default value specifies that this variable will match any taint on any node except the master node. For more information about tolerations and taints, see [Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)."
```

```suggestion
  description = "The list of tolerations to apply to the IBM Cloud Monitoring agent. The default value matches any taint on any node except the master node. [Learn more](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)"
```

Suggest that we go shorter because of space issues. Also the variable isn't matched, the toleration is, but I think it is implied.